src/features/workflow/modal/DeployModal.tsx

```diff
@@ -177,12 +177,18 @@ const DeployModal: React.FC<DeployModalProps> = () => {
         <Button
           size='sm'
           type='secondary'
-          className='w-full mt-2'
+          className='w-full mt-2 flex-shrink-0'
           onClick={handleDeployClick}
           submit
           disabled={!readyToDeploy}>
           <Icon icon='rocket' className='mr-2' /> Deploy
         </Button>
+        {workflow.triggers?.length === 0 && (
+          <div className='flex-shrink-0 text-qrigray-400 text-xs border-2 border-warningyellow rounded mt-5 mb-5 py-1.5 px-2 leading-tight flex'>
+            <div className='mr-2 text-warningyellow'><Icon icon='circleWarning' size='sm'/></div>
+            <div>This workflow does not have any triggers defined.  Once deployed, it will only run when triggered manually.</div>
+          </div>
```

Comment:
factor this into a new `WarningDialog` component in `chrome/WarningDialog.tsx`. give it a `text` prop with the no triggers message.